guides/source/caching_with_rails.md

```diff
@@ -88,21 +88,19 @@ When your application receives its first request to this page, Rails will write
 a new cache entry with a unique key. A key looks something like this:
 
 ```
-views/products/1-201505056193031061005000/bea67108094918eeba42cd4a6e786901
+views/products/index.html.erb:bea67108094918eeba42cd4a6e786901/products/1
```

Comment:
The template format isn't part of the cache key, rather it's included in the digest:

https://github.com/rails/rails/blob/5d4c74557c1e9609ee7ff0c3a24658e3a75be441/actionview/lib/action_view/helpers/cache_helper.rb#L220

```suggestion
views/products/index:bea67108094918eeba42cd4a6e786901/products/1
```

Reply:
@eugeneius hmm if that's correct then this comment https://github.com/rails/rails/blob/master/actionview/lib/action_view/helpers/cache_helper.rb#L27 should be updated too, right?